packages/storybook/src/stories/ino-radio/ino-radio.stories.ts

```diff
@@ -14,56 +15,57 @@ export default {
     },
   },
   decorators: [
-    (story) => decorateStoryWithClass(story, 'story-radio'),
-    (story) => {
+    story => decorateStoryWithClass(story, 'story-radio'),
+    story => {
       useEffect(() => {
-        const eventHandler = (e) => e.target.setAttribute('checked', e.detail);
+        const eventHandler = e => e.target.setAttribute('checked', e.detail);
         const radios = document.querySelectorAll('ino-radio');
-        radios.forEach((r) =>
-          r.addEventListener('checkedChange', eventHandler)
-        );
+        radios.forEach(r => r.addEventListener('checkedChange', eventHandler));
         return () =>
-          radios.forEach((r) =>
+          radios.forEach(r =>
             r.removeEventListener('checkedChange', eventHandler)
           );
       });
       return story();
     },
   ],
-} as Meta;
+  args: {
+    checked: false,
+    disabled: false,
+    name: 'radio-custom',
+    value: 'radio-1',
+  },
+} as Meta<Components.InoRadio>;
 
-export const Playground: Story<Components.InoRadio> = (args) => html`
-  <ino-radio
-    checked="${args.checked}"
-    disabled="${args.disabled}"
-    name="${args.name}"
-    value="${args.value}"
-  >
-    Radio Button Label
-  </ino-radio>
-`;
-Playground.args = {
-  checked: false,
-  disabled: false,
-  name: 'radio-custom',
-  value: 'radio-1',
-};
+const template = new TemplateGenerator<Components.InoRadio>(
+  'ino-radio',
+  args => html`
+    <ino-radio
+      checked="${args.checked}"
+      disabled="${args.disabled}"
+      name="${args.name}"
+      value="${args.value}"
+    >
+      Radio Button Label
+    </ino-radio>
+  `
```

Comment:
In case of using the template string the intends generates an not well formated example code.

![image](https://user-images.githubusercontent.com/23154336/167784714-7156070f-12a7-4558-a727-60ebe11b0c47.png)

```suggestion
  args => html`
  <ino-radio
    checked="${args.checked}"
    disabled="${args.disabled}"
    name="${args.name}"
    value="${args.value}"
  >
    Radio Button Label
  </ino-radio>
`
```

Reply:
https://github.com/storybookjs/storybook/issues/12022#issuecomment-859540813 Can this help?

Reply:
Yes, it's working as described but disabling the decorators also disables all of our event handlers as they are also implemented as decorators